app/src/main/kotlin/cz/covid19cz/erouska/db/SharedPrefsRepository.kt

```diff
@@ -65,6 +67,14 @@ class SharedPrefsRepository(c : Context) {
         return prefs.getLong(LAST_UPLOAD_TIMESTAMP, -1)
     }
 
+    fun saveAuthPhoneNumber(phoneNumber: String) {
+        prefs.edit().putString(AUTH_PHONE_NUMBER, "-").apply()
```

Comment:
pockej vzdyt takhle v tech shared prefs nebude nikdy nic jinyho nez pomlcka? 
Asi bych to udelal tak, ze pri loginu ulozim cislo do shared prefs a cetl bych ho primarne ze shared prefs.
Pokud tam nebude, tak zkusim nacist z firebase (a updatovat v shared prefs) a pokud ani tam ne, tak vratit -..
Klidne muzeme otocit to prioritu firebase a shared prefs..to by mozna davalo vetsi smysl a mit ty jenom jako backup.